src/shared/bus-polkit.h

```diff
@@ -12,6 +12,7 @@ typedef enum PolkitFLags {
         POLKIT_ALWAYS_QUERY      = 1 << 1, /* Query polkit even if client is privileged */
         POLKIT_DEFAULT_ALLOW     = 1 << 2, /* If polkit is not around, assume "allow" rather than the usual "deny" */
         POLKIT_DONT_REPLY        = 1 << 3, /* Varlink: don't immediately propagate polkit error to the Varlink client */
+        _POLKIT_MASK_PUBLIC      = POLKIT_ALLOW_INTERACTIVE | POLKIT_ALWAYS_QUERY, /* polkit accepts these flags verbatim */
 } PolkitFlags;
```

Comment:
Hmm, if you touch this, could you fix the name of the enum? (PolkitFLags -> Flags)

Reply:
Isn't "Flags" too generic?

Reply:
(I'm referring to `typedef enum PolkitFLags` above)

Reply:
ah the typo - didn't even see it